components/dot_menu/index.ts

```diff
@@ -122,7 +123,7 @@ function mapStateToProps(state: GlobalState, ownProps: Props) {
         pluginMenuItems: state.plugins.components.PostDropdownMenu,
         canEdit: PostUtils.canEditPost(state, post, license, config, channel, userId),
         canDelete: PostUtils.canDeletePost(state, post, channel),
-        currentTeamUrl,
+        teamUrl,
         currentTeamId: currentTeam.id,
```

Comment:
We're still using the current team ID for some stuff in the component like following threads which seem like they might break when viewing messages on another team.

Perhaps instead of passing the URL around, we should just pass the team ID around? That may not work for posts in DMs though, so feel free to ignore that part